common/src/main/java/org/figuramc/figura/animation/Animation.java

```diff
@@ -140,6 +141,25 @@ public void playCode(float minTime, float maxTime) {
     // -- lua methods -- // 
 
 
+    @LuaWhitelist
+    @LuaMethodDoc("animation.getChannels")
+    public LuaTable getChannels() {
+        LuaTable ret = new LuaTable();
+        for (Map.Entry<FiguraModelPart, List<Animation.AnimationChannel>> entry : animationParts) {
+            FiguraModelPart part = entry.getKey();
+            List<Animation.AnimationChannel> value = new ArrayList<Animation.AnimationChannel>();
+            for (Animation.AnimationChannel channel : entry.getValue()) {
+                value.add(channel);
+            }
```

Comment:
Is there any need to iterate over all channels instead of initializing the list with the collection?

Reply:
idk